src/main/java/com/example/Restaurant_Management_System_REST_API/model/entity/Customer.java

```diff
@@ -34,15 +35,18 @@ public class Customer implements UserDetails {
     private ContactDetails contactDetails;
 
     //Below are fields for security issues
-    private String password;
+    private String password; //validation for this field is provided in CustomerDTORequest
     private Boolean accountNonExpired;//to avoid hardcoding I established these fields
     private Boolean accountNonLocked;
     private Boolean credentialsNonExpired;
     private Boolean enabled;
+    @NotBlank(message = "Email is missing!")
+    @Email(regexp=".+@.+\\..+", message = "Email address must be as email: example@example.pl")
```

Comment:
Would be nicer to create a message that would be more meaningful, more understandable. The wording of that message means that email **must be** the example you provided.